File: scala/b02.scala
import  scala.io.StdIn._

object Main {
    def main(args: Array[String]): Unit= {
        val Array(a, b) = readLine().split(" ").map(_.toInt)
        val isFound = (a to b).map(100 % _).filter(_ == 0).length
        val ans = if (isFound > 0) "Yes" else "No"
        println(ans)
    }
}

File: rust/b02.rs
use proconio::input;

fn main() {
    input! {
        a: u32,
        b: u32,
    }
    let mut is_found: bool = false;
    for i in a..=b {
        if 100 % i == 0 {
            is_found = true;
            break;
        }
    }
    if is_found {
        println!("Yes")
    } else {
        println!("No")
    }
}
